physics/test/BasicIceOceanFlux_test.cpp

```diff
@@ -130,12 +130,12 @@ TEST_CASE("Freezing conditions", "[BasicIceOceanHeatFlux]")
     public:
```

Comment:
Suspect this class can be brought out of the case and used across cases (or test files)